src/coreclr/gc/gcinterface.dacvars.def

```diff
@@ -68,6 +79,11 @@ GC_DAC_PTR_VAR    (uint8_t,               bookkeeping_start)
 GC_DAC_ARRAY_VAR  (dac_region_free_list,  global_regions_to_decommit)
 GC_DAC_PTR_VAR    (dac_region_free_list,  global_free_huge_regions)
 GC_DAC_ARRAY_VAR  (dac_region_free_list,  free_regions)
+GC_DAC_PTR_VAR    (dac_heap_segment,      freeable_soh_segment)
```

Comment:
Is this the start of the minor version 2 fields? I'd suggest inserting a comment saying that. It helps document how the contract changed in each version.